src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -688,10 +688,6 @@ private function getPropertyName(string $methodName, array $reflectionProperties
      */
     private function findAdderAndRemover(\ReflectionClass $reflClass, array $singulars): array
     {
-        if (!\is_array($this->arrayMutatorPrefixes) && 2 !== \count($this->arrayMutatorPrefixes)) {
```

Comment:
why don't we need the `2 !== \count($this->arrayMutatorPrefixes)` check anymore?

Reply:
being a public static method, I would keep the safeguard here as there is nothing avoiding mutating it, even though it is internal.

Reply:
IIUC, this means this change should be reverted, isn't it @alamirault?

Reply:
I reverted `2 !== \count($this->arrayMutatorPrefixes)` part